### PR TITLE
chore: bump release-service-utils image in rh-sign-image-cosigin

### PR DESCRIPTION
**Pull request by jinqi7**

### CODE REVIEW

File: tasks/rh-sign-image-cosign/README.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ Tekton task to sign container images in snapshot by cosign.
 | secretName             | Name of secret containing needed credentials                                                                                                                                                                                                      | No       | -             |
 | signRegistryAccessPath | The relative path in the workspace to a text file that contains a list of repositories that needs registry.access.redhat.com image references to be signed (i.e. requires_terms=true), one repository string per line, e.g. "rhtas/cosign-rhel9". | No       | -             |
 
+## Changes in 1.1.0
+* Bump release-service-utils to upgrade cosign version for fixing the idempotency error in signing.
```
Comment (mmalina) [inline tasks/rh-sign-image-cosign/README.md]: Can you specify the version? Perhaps something like this:
```suggestion
* Bump release-service-utils to upgrade cosign version to 2.4.0 which should fix an issue the osci team is having when trying to release.
```

Comment (jinqi7) [reply tasks/rh-sign-image-cosign/README.md]: Updated. Thanks

**Review by mmalina — APPROVED** [review]

File: tasks/rh-sign-image-cosign/README.md (hunk 1 above)
Comment (mmalina) [reply tasks/rh-sign-image-cosign/README.md]: Cool. E2E signing should be fixed now. Let's see if it works.

**Pull request merged**